recipes/default.rb

```diff
@@ -24,7 +24,7 @@
 # WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 #
 
-include_recipe 'python'
+python_runtime 'beaver'
```

Comment:
That's not quite how I interpreted that statement. While it probably can work from a wrapper, I think it's intended to be a shortcut within the same recipe. All `python_runtime` does is install some version from some source and the `beaver` bit means you can easily state which version and which source for this particular context via attributes. It allows you to install a different version from a different source in a different context for some other application. These may well end up using the same Python installation but either way, that's fine.

> Setting via node attributes is generally how an end-user or application cookbook will set options to customize installations in the library cookbooks they are using.

Is there anything you need to do that can't be done via the attributes?